### DISCUSSION TITLE
Wellness Blog in Community Newsletter

### DISCUSSION THREAD

**Discussion by SaraJaoude** [EddieHubCommunity/support#1895] [Ideas]
What topics would the Community like to read about in the Wellness blog, to help them make sure they have down time, self care and do not burn out?

**Comment by kaiwalyakoparkar** [comment]
How to utilize downtime effectively!

**Comment by mikeysan** [comment]
> What topics would the Community like to read about in the Wellness blog, to help them make sure they have down time, self care and do not burn out?

In a typical 8 hour work day, how often should I step away from my computer?

**Comment by SaraJaoude** [comment]
@kaiwalyakoparkar and @mikeysan  - these are really great topics, thank you! Hope you are both managing to get some downtime :)
